.env.production

```diff
@@ -0,0 +1,21 @@
+# Production Environment Configuration
+MOVICI_FLOW_DATA_DIR=/app/data
+MOVICI_FLOW_USE_GLOBAL_PLUGINS=true
+MOVICI_FLOW_VALIDATE_UPDATES=true
+
+# Logging
+LOG_LEVEL=INFO
+LOG_FORMAT=json
+
+# Security
+CORS_ORIGINS=https://yourdomain.com
+API_RATE_LIMIT=100
+
+# Performance
+WORKERS=4
+MAX_CONNECTIONS=1000
+KEEPALIVE_TIMEOUT=5
+
+# Monitoring
+ENABLE_METRICS=true
+METRICS_PORT=9090
```

Comment:
Movici view does not have a production environment. Also some of the values here do not make any sense